Tasks/powerShell/task.json

```diff
@@ -50,13 +64,25 @@
             "required":false,
             "helpMarkDown": "Current working directory when script is run.  Defaults to the folder where the script is located.",
             "groupName":"advanced"
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "defaultValue": "Write-Host 'Hello World'",
+            "required": true,
+            "helpMarkDown": "Write your scripts and have fun",
+            "visibleRule": "type = InlineScript",
+            "properties": {
+                "rows": 5
+            }
         }
     ],
     "instanceNameFormat": "Powershell: $(scriptName)",
     "execution": {
-        "PowerShellExe": {
-            "target": "$(scriptName)",
-            "argumentFormat": "$(arguments)",
+        "PowerShell": {
```

Comment:
@ericsciple I will revert to using the PowerShellExe handler. Thanks for letting me know about the problems that could happen because of the PS handler. I wasn't aware of the current mess with this task.